ops/vars/ansistrano.yml

```diff
@@ -13,7 +13,8 @@ ansistrano_shared_paths:
 
 ansistrano_deploy_via: "git"
 ansistrano_git_repo: https://github.com/ucsdlib/horton-bot.git
-ansistrano_git_branch: master
+# TODO: change to master before deployment.
+ansistrano_git_branch: ansibilize
```

Comment:
Pretty sure `ansistrano_git_branch` will need to change _before_  the PR gets merged